docs/sources/flow/reference/cli/convert.md

```diff
@@ -80,6 +81,8 @@ This includes Prometheus features such as
 and many supported *_sd_configs. Unsupported features in a source config result
 in [errors].
 
+You can find a detailed migration guide for this [here](../../../getting-started/migrating-from-prometheus/)
```

Comment:
```suggestion
Refer to [Migrate from Prometheus to Grafana Agent Flow]({{< relref "../../getting-started/migrating-from-prometheus/" >}}) for a detailed migration guide.
```

We need to double check I've got the number of `../` correct in the relref.  Can be done in a local checkout with `make docs` and check links, or with `make doc-validator`.